File: sandbox/ruby/src/TeamInfoDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::TeamApi.new.team_info(
        {
            team_id: "4fea99bfcf2b26bfccf6cea3e127fb8bb74d8d9c",
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Team#team_info: #{e}"
end

File: sandbox/node/src/TeamInfoDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TeamApi();

const teamId = "4fea99bfcf2b26bfccf6cea3e127fb8bb74d8d9c";

apiCaller.teamInfo(
    teamId,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Team#teamInfo:");
  console.log(error.body);
});
